shared/instance.go

```diff
@@ -245,6 +237,14 @@ var InstanceConfigKeysContainer = map[string]func(value string) error{
 	"security.syscalls.intercept.setxattr":           validate.Optional(validate.IsBool),
 	"security.syscalls.intercept.sysinfo":            validate.Optional(validate.IsBool),
 	"security.syscalls.whitelist":                    validate.IsAny,
+
+	"volatile.last_state.idmap": validate.IsAny,
+	"volatile.idmap.base":       validate.IsAny,
+	"volatile.idmap.current":    validate.IsAny,
+	"volatile.idmap.next":       validate.IsAny,
+
+	// Caller is responsible for full validation of any raw.* value.
+	"raw.idmap": validate.IsAny,
```

Comment:
That one is wrong as it's also used for VMs when passing in shared directories from the host inside of a restricted project.

Reply:
ah yes, for some reason I thought we had that on the disk device's config rather than on the instance itself.

Reply:
Fixed